src/content/resources/glossary.md

```diff
@@ -336,3 +336,68 @@ class A implements B {} // A is a subtype of B, but NOT a subclass of B.
 
 class C extends D {} // C is a subtype AND a subclass of D.
 ```
+
+## Variance and variance positions
+
+A type parameter of a class (or other type declaration, like a mixin) is
+said to be _covariant_ when the type as a whole "co-varies" with the actual
+type argument. In other words, if the type argument is replaced by a
+subtype then the type as a whole is also a subtype.
+
+For example, the type parameter of the class `List` is covariant because
+list types co-vary with their type argument: `List<int>` is a subtype of
+`List<Object>` because `int` is a subtype of `Object`.
+
+In Dart, all type parameters of all class, mixin, mixin class, and enum
+declarations are covariant.
+
+However, function types are different: A function type is covariant in its
+return type, but the opposite (known as _contravariant_) in its parameter
+types.  For example, the type `int Function(int)` is a subtype of the type
+`Object Function(int)`, but it is a supertype of `int Function(Object)`.
+
+This makes sense if you consider their [substitutability](#subtype). If you
+call a function with a static type of `int Function(int)`, that function
+can actually be of type `int Function(Object)` at runtime. Based on the
+static type, you expect to be able to pass an `int` to it. That will be
+fine since the function actually accepts any `Object`, and this includes
+every object of type `int`. Similarly, the returned result will be of type
+`int`, which is also what you expect based on the static type.
+
+Hence, `int Function(Object)` is a subtype of `int Function(int)`.
+
+Note that everything is turned upside-down for parameter types. In
+particular, this subtype relation among function types requires that the
+_opposite_ subtype relation exists for the parameter type (for example,
+`void Function(Object)` is a subtype of `void Function(int)` because `int`
+is a subtype of `Object`).
+
+With a more complex type like `List<void Function(int)>`, you have to
+consider the _positions_ in the type. This is basically just a matter of
+turning one of the parts of the type into a placeholder (let's use `_` for
+that), and then consider what happens to the type when different types are
```

Comment:
Generally it's best to avoid phrasing like "basically just" as it don't add much meaning, and this process might not be basic for some readers. https://developers.google.com/style/word-list#simple

How about we be more prescriptive in how to do this:

```suggestion
consider the _positions_ in the type. To accomplish this,
turn one of the parts of the type into a placeholder,
and then consider what happens to the type when different types are
```

In this suggestion I also proposed dropping the "(let's use `_` for that)" to keep this sentence shorter and focused. The next paragraph can introduce that choice.

Reply:
Great point, thanks!